stable/firmament-poseidon/templates/firmament_service.yaml

```diff
@@ -0,0 +1,18 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: firmament-service
+  namespace: {{ .Values.namespace }}
+  labels:
+   app: {{ template "firmament-poseidon.name" . }}
+   chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+   heritage: {{ .Release.Service }}
+   release: {{ .Release.Name }}
+   scheduler: {{ .Values.firmament.scheduler }}
+spec:
+  selector:
+    scheduler: firmament
```

Comment:
While you make the label configurable in the deployment, you hard-code it here. Common practice is to use a `component` label. Add `app` and `release` labels.